src/processing/resources/page.js

```diff
@@ -67,13 +68,13 @@ class PageProcessor extends ResourceProcessorBase {
         var resources = [];
 
         if (processingOptions.styleUrl) {
-            resources.push(`<link rel="stylesheet" type="text/css" class="${Const.SHADOW_UI_STYLESHEET_FULL_CLASSNAME}"
+            resources.push(dedent`<link rel="stylesheet" type="text/css" class="${Const.SHADOW_UI_STYLESHEET_FULL_CLASSNAME}"
```

Comment:
Don't use dedent, just concat to avoid new line in resulting code (same goes for the next script injection):

```js
resources.push(
    `<link rel="stylesheet" type="text/css" class="${Const.SHADOW_UI_STYLESHEET_FULL_CLASSNAME}"` + 
    `href="${processingOptions.styleUrl}">`
);
```